Automattic/crowdsignal-forms
# Add keywords and example

This PR adds

  - some keywords for discovery on the block search
  - the example to be shown on the block search
  - the block icon

The example demanded an extra attribute `isExample` so we could prevent the triggering of the autosaving function.

![image](https://user-images.githubusercontent.com/157240/107804576-ca0c3980-6d42-11eb-902d-5341329c5238.png)


Suggestions are welcome.

## Test instructions
Checkout and `make client`. Use the block search with any of the keywords. See the example renders on mouse over.
